ui/page.go

```diff
@@ -687,6 +693,32 @@ type SubPage struct {
 }
 
 func (page pageCommon) SubPageLayout(gtx layout.Context, sp SubPage) layout.Dimensions {
+	return page.theme.Card().Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D { return page.subPageHeader(gtx, &sp) }),
+			layout.Rigid(func(gtx layout.Context) layout.Dimensions {
+				return layout.Inset{
+					Left:   values.MarginPadding15,
+					Right:  values.MarginPadding15,
+					Bottom: values.MarginPadding15,
+				}.Layout(gtx, sp.body)
+			}),
+		)
+	})
+}
+
+func (page pageCommon) SubpageSplitLayout(gtx layout.Context, sp SubPage) layout.Dimensions {
+	card := page.theme.Card()
+	card.Color = color.RGBA{}
+	return card.Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D { return page.subPageHeader(gtx, &sp) }),
+			layout.Rigid(sp.body),
+		)
+	})
+}
+
+func (page pageCommon) subPageHeader(gtx layout.Context, sp *SubPage) layout.Dimensions {
```

Comment:
Why's this method taking a pointer as an argument? I don't see anywhere in the code where a property is being set.